frontend/debian/cuttlefish-orchestration.postinst

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+    (configure)
```

Comment:
shouldn't it be `configure)`?
Also, a `switch` with only one case should probably be an `if`.

Reply:
it seems the open parenthesis is optional, I copied/pasted from other postint script without noticing.

I think is better to keep the `case` statement as it's kind of standard in `postint` scripts making easier the future job of managing new options.

Reply:
SG